Subject: Adjust phpMyAdmin vendor configuration to match Debian needs
 - configuration is in /etc/phpmyadmin-omnibus
 - documentation is in /usr/share/doc
 - config file consists of several included files, so we skip mtime check
 - tempdir is in /var/lib/phpmyadmin-omnibus/tmp
From: Michal Čihař <nijel@debian.org>
Forwarded: not-needed
--- a/libraries/vendor_config.php
+++ b/libraries/vendor_config.php
@@ -22,19 +22,19 @@
 /**
  * Directory where cache files are stored.
  */
-define('TEMP_DIR', './tmp/');
+define('TEMP_DIR', '/var/lib/phpmyadmin-omnibus/tmp/');
 
 /**
  * Path to changelog file, can be gzip compressed. Useful when you want to
  * have documentation somewhere else, eg. /usr/share/doc.
  */
-define('CHANGELOG_FILE', './ChangeLog');
+define('CHANGELOG_FILE', '/usr/share/doc/phpmyadmin-omnibus/changelog.gz');
 
 /**
  * Path to license file. Useful when you want to have documentation somewhere
  * else, eg. /usr/share/doc.
  */
-define('LICENSE_FILE', './LICENSE');
+define('LICENSE_FILE', '/usr/share/doc/phpmyadmin-omnibus/copyright');
 
 /**
  * Directory where SQL scripts to create/upgrade configuration storage reside.
@@ -46,7 +46,7 @@
  * It is not used directly in code, just a convenient
  * define used further in this file.
  */
-define('CONFIG_DIR', '');
+define('CONFIG_DIR', '/etc/phpmyadmin-omnibus/');
 
 /**
  * Filename of a configuration file.
@@ -66,7 +66,7 @@
 /**
  * Default value for check for version upgrades.
  */
-define('VERSION_CHECK_DEFAULT', true);
+define('VERSION_CHECK_DEFAULT', false);
 
 /**
  * Path to files with compiled locales (*.mo)
